service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -107,16 +92,8 @@ public CaseDataExtractionService(DateFormatterService dateFormatterService,
 
         if (isNotEmpty(caseData.getStandardDirectionOrder())
             && caseData.getStandardDirectionOrder().getOrderStatus() != SEALED) {
-            byte[] fileContent;
-            try {
-                InputStream is = getClass().getResourceAsStream("/assets/images/draft-watermark.png");
-                fileContent = is.readAllBytes();
-                String encodedString = Base64.getEncoder().encodeToString(fileContent);
-
-                data.put("draftbackground", String.format("image:base64:%1$s", encodedString));
-            } catch (IOException e) {
-                log.warn(e.getMessage());
-            }
+            data.put("draftbackground", String.format("image:base64:%1$s",
```

Comment:
Good idea to move it although I wonder what case data extraction (since name of the service) has to do with watermarks? Looks like bad naming or bad abstraction :(

Reply:
its currently used within the `getStandardOrderDirectionData` method (an existing method) and we just moved the code out to make more reusable instead of duplicating.

Reply:
Moved to a new abstract class that future docmosis template data providers can then extend

Reply:
However I have left it here for CaseDataExtraction service so as to not change to much, future refactor